bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: correct format of data plane observability config sample

## Description

There were mixed whitespaces and tabs. The set of changes make sure that only whitespaces are used.

## Verification Steps

* Verify indentation in the file is correct.
* Verify the YAML is syntactically valid

## Checklist (Definition of Done)
